### PR TITLE
Add gazebo9 source package import formula (Bionic)

### PR DESCRIPTION
**Pull request by j-rivero**
The formula will import the gazebo source package corresponding to the packages in http://repos.ros.org/repos/ros_bootstrap/pool/main/g/gazebo/.

There is an ancient 1.9 source package named gazebo there that can be overwritten safely since all versions using it are EOL.

### CODE REVIEW

File: config/gazebo9_bionic.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: gazebo9bionic
+method: https://launchpad.net/~j-rivero/+archive/ubuntu/gazebo9bionic
```
Comment (nuclearsandwich) [inline config/gazebo9_bionic.yaml:2]:
```suggestion
method: http://ppa.launchpad.net/j-rivero/gazebo9bionic/ubuntu
```

The method needs to be the apt repository URL not the web link to the PPA.

File: config/gazebo9_bionic.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+suites: [bionic]
+component: main
+architectures: [source]
```
Comment (nuclearsandwich) [inline config/gazebo9_bionic.yaml:5]: Do you think it makes sense to import the binary packages as well in order to ensure that we're soundly using packages generated from this source package? That's what we did for pyside2 and I think it makes sense to do here as well.

File: config/gazebo9_bionic.yaml (hunk 1 above)
Comment (j-rivero) [reply config/gazebo9_bionic.yaml:2]: ouch, thanks.

**Review by nuclearsandwich — APPROVED** [review]

File: config/gazebo9_bionic.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+name: gazebo9bionic
+method: http://ppa.launchpad.net/j-rivero/gazebo9bionic/ubuntu
+suites: [bionic]
+component: main
+architectures: [amd64, arm64, armhf, i386, source]
+filter_formula: Package (% gazebo9), $Version (% 9.0.0+dfsg5-3ubuntu1+ppa2)
```
Comment (nuclearsandwich) [inline config/gazebo9_bionic.yaml:6]: Do we need the libgazebo9* packages as well? Would you accept a change that uses exact package names for clarity, matching the format used in #94?

Comment (j-rivero) [reply config/gazebo9_bionic.yaml]: 2e1b8d0

**Review by nuclearsandwich — APPROVED** [review]

**Comment by nuclearsandwich** [comment]
Import jobs
https://build.ros.org/job/import_upstream/156/
https://build.ros2.org/job/import_upstream/309/

**Pull request merged**